lib/stimulus_reflex/reflex.rb

```diff
@@ -152,4 +119,45 @@ def render_collection(resource, content = nil)
     content ||= render(resource)
     tag.div(content.html_safe, id: dom_id(resource).from(1))
   end
+
+  private
+
+  def build_request
+    begin
+      uri = URI.parse(url)
```

Comment:
Note: in this context,[`self.url`](https://github.com/nakwa/stimulus_reflex/pull/1/files#diff-87b7a54fdec6e28f9dee9180a91acf18c74c073940340e9e6d56b5fcb9679c40R12) could be changed from `attr_reader` to `attr_accessor` , so that it can be changed after the reflex is initialized. 

Alternatively, considering it may not make sense to change the `path` itself, but only querystring parameters, there could be a setter method for pushing new query to the path, during the reflex execution, 
e.g. 
```
    def push_query(next_query = {})
      uri = URI.parse(request.fullpath)
      path = ActionDispatch::Journey::Router::Utils.normalize_path(uri.path)
      current_query = Rack::Utils.parse_nested_query(uri.query)
      query_hash = ActiveSupport::HashWithIndifferentAccess.new({ **current_query, **next_query })
      @url = "#{path}?#{query_hash.to_query}"
    end
```